src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -77,27 +77,32 @@ private static void ValidateAndRoundtrip(object obj, string[] blobs, bool isEqua
             // Therefore we remove the second blob which is the one from Desktop.
             if (!PlatformDetection.IsFullFramework && (obj is SqlException || obj is ReflectionTypeLoadException || obj is LicenseException))
             {
-                var tmpList = new List<string>(blobs);
+                var tmpList = new List<TypeSerializableValue>(blobs);
                 tmpList.RemoveAt(1);
+
+                int index = tmpList.FindIndex(b => b.Platform == TargetFrameworkMoniker.netfx);
```

Comment:
this should be `b.Platform == TargetFrameworkMoniker.netfx || b.Platform == TargetFrameworkMoniker.netfx471)`